internal/service/streamprocessor/data_source_plural_schema.go

```diff
@@ -0,0 +1,51 @@
+package streamprocessor
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+)
+
+var _ datasource.DataSource = &StreamProccesorDS{}
+var _ datasource.DataSourceWithConfigure = &StreamProccesorDS{}
+
+func PluralDataSource() datasource.DataSource {
+	return &streamProcessorsDS{
+		DSCommon: config.DSCommon{
+			DataSourceName: fmt.Sprintf("%ss", StreamProcessorName),
+		},
+	}
+}
+
+type streamProcessorsDS struct {
+	config.DSCommon
+}
+
+func (d *streamProcessorsDS) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"project_id": schema.StringAttribute{
+				Required: true,
+			},
+			"instance_name": schema.StringAttribute{
+				Required: true,
+			},
```

Comment:
add description?

Reply:
Thanks! Added in [6902084](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2505/commits/69020842dd342fffa6fdb7cd060ef6e1dce18d08)